lib/utils/atRuleParamIndex.mjs

```diff
@@ -1,10 +1,11 @@
+import atRuleRawsAfterNameIndex from './atRuleRawsAfterNameIndex.mjs';
+
 /**
  * @param {import('postcss').AtRule} atRule
  * @returns {number}
  */
 export default function atRuleParamIndex(atRule) {
```

Comment:
minor
question

`atRuleParamIndex` is used by 14 other rules.
I suppose this modification fixes a bug hence the changeset could have 14 rules except `selector-pseudo-class-no-unknown` which already has its own changeset in this release.